tests/conftest.py

```diff
@@ -183,7 +183,7 @@ def fillna(data: pd.DataFrame):
             'weird_feature': np.random.choice(np.array([1, 100, 1.0, 'ahh?', 'wee', np.nan, 0],
                                               dtype='object'), size=1000),
             8: pd.array(np.random.choice([0, 1, 5, 6, np.nan], size=1000), dtype='Int64'),
-            'tuples': np.random.choice([(0, 2), (1, 6, 8), (9, 1), (8, 1, 9, 8)], size=1000),
+            'tuples': random.choices([(0, 2), (1, 6, 8), (9, 1), (8, 1, 9, 8)], k=1000),
```

Comment:
does this work on older numpy versions as well?
we can also do np.random.choice(np.asarray([(0, 2), (1, 6, 8), (9, 1), (8, 1, 9, 8)], dtype='object'), size=1000)

Reply:
I didn't really understand the suggestion, on new numpy it raises error because the items inside the array are in different lengths from each other